spec/expression.dd

```diff
@@ -21,13 +21,47 @@ full expression of $(I expr$(SUBSCRIPT 1)). Otherwise, $(I expr) is its own full
 
 $(P Each expression has a unique full expression.)
 
-Example: in the statement `return f() + g() * 2;`, the full expression of `g() * 2` is `f() + g() * 2`, but not the
-full expression of `f() + g()` because the latter is not parsed as a subexpression.
+$(P Example: in the statement `return f() + g() * 2;`, the full expression of `g() * 2` is `f() + g() * 2`, but not the
+full expression of `f() + g()` because the latter is not parsed as a subexpression.)
 
-Note: Although the definition is straightforward, a few subtleties exist related to function literals. In the
+$(P Note: Although the definition is straightforward, a few subtleties exist related to function literals. In the
 statement `return (() => x + f())() * g();`, the full expression of `f()` is `x + f()`, not the expression passed
-to `return`. This is because the parent of `x + f()` has function literal type, not expression type.
-
+to `return`. This is because the parent of `x + f()` has function literal type, not expression type.)
+
+$(DDOC_ANCHOR define-lvalue)$(P $(B Definition) ($(DOUBLEQUOTE Lvalue)): The following expressions, and no
+others, are called lvalue expressions or lvalues:)
+$(OL
+$(LI `this` inside `struct` and `union` member functions;)
+$(LI a simple symbol or the result of the $(I DotIdentifier) grammatical construct `.` (left side may be
```

Comment:
`simple symbol` is not defined. It's erroneous anyway as a "simple symbol" that is a manifest constant is not an lvalue. I suggest replacing it with `variable`.